### PR TITLE
Active page/item disappears for few items

### PR DESCRIPTION
**Pull request by Mirodil**
fix issue for less items, see #2270

### CODE REVIEW

File: src/Pagination.js [hunk 1]
```diff
@@ -87,8 +87,8 @@ class Pagination extends React.Component {
 
     if (maxButtons) {
       let hiddenPagesBefore = activePage - parseInt(maxButtons / 2, 10);
-      startPage = hiddenPagesBefore > 2 ? hiddenPagesBefore : 1;
-      hasHiddenPagesAfter = startPage + maxButtons < items;
+      startPage = hiddenPagesBefore >= 2 ? hiddenPagesBefore : 1;
```
Comment (taion) [inline src/Pagination.js]: `Math.max`?

File: src/Pagination.js [hunk 2]
```diff
@@ -87,8 +87,8 @@ class Pagination extends React.Component {
...
+      hasHiddenPagesAfter = startPage + maxButtons <= items;
```
Comment (taion) [inline src/Pagination.js]: would be clearer as `items >= ...`

File: src/Pagination.js (hunk 1 above)
Comment (Mirodil) [inline src/Pagination.js]: Where are you going to add `Math.max`?

File: src/Pagination.js (hunk 2 above)
Comment (Mirodil) [inline src/Pagination.js]: I tried to change less code

**Comment by Mirodil** [comment]
```
  it('should render three items from 1 to 3 when activePage=1', () => {
    const instance = ReactTestUtils.renderIntoDocument(
      <Pagination items={4} maxButtons={3} activePage={1} />
    );
    const pageButtons = ReactTestUtils.scryRenderedDOMComponentsWithTag(instance, 'li');

    assert.equal(pageButtons[0].textContent, '1');
    assert.equal(pageButtons[1].textContent, '2');
    assert.equal(pageButtons[2].textContent, '3');
  });

  it('should render three items from 2 to 4 when activePage=3', () => {
    const instance = ReactTestUtils.renderIntoDocument(
      <Pagination
        onSelect={()=>{}}
        items={4}
        maxButtons={3}
        activePage={3} />
    );
    const pageButtons = ReactTestUtils.scryRenderedDOMComponentsWithTag(instance, 'li');

    assert.equal(pageButtons[0].textContent, '2');
    assert.equal(pageButtons[1].textContent, '3');
    assert.equal(pageButtons[2].textContent, '4');
  });
```

**Comment by Mirodil** [comment]
done

File: src/Pagination.js (hunk 1 above)
Comment (taion) [inline src/Pagination.js]: yes

File: src/Pagination.js (hunk 2 above)
Comment (taion) [inline src/Pagination.js]: I think the clarity of making it `items >= ...` is worthwhile

**Comment by taion** [comment]
Please address the PR comments. Otherwise this looks fine to m.e

**Comment by taion** [comment]
I went ahead and made the changes.

**Comment by jquense** [comment]
lgtm

**Pull request merged**